docs/usage/deep-linking.md

```diff
@@ -1,19 +1,20 @@
 # `deepLinking` parameter
 
-Swagger-UI allows you to deeply link into tags and operations within a spec. When Swagger-UI is provided a URL fragment at runtime, it will automatically expand and scroll to a specified tag or operation.
+Swagger-UI allows you to deeply link into tags and operations within a spec. When a deep link is clicked, it will automatically expand and scroll to a specified tag or operation.
 
 ## Usage
 
-👉🏼 Add `deepLinking: true` to your Swagger-UI configuration to enable this functionality.
+👉🏼 Add `deeplinking: true` to your Swagger-UI configuration to enable this functionality.
 
-When you expand a tag or operation, Swagger-UI will automatically update its URL fragment with a deep link to the item.
-Conversely, when you collapse a tag or operation, Swagger-UI will clear the URL fragment.
+To create a deep link in your spec, **only use** an anchor element whose name attribute is assigned `"deeplink"` (if this value is not assigned your deep link will not work) **and** whose href attribute is assigned a hash.
+
+When you expand a tag or operation, Swagger-UI will automatically update its URL fragment with that item's hash. Conversely, when you collapse a tag or operation, Swagger-UI will clear the URL fragment.
 
 You can also right-click a tag name or operation path in order to copy a link to that tag or operation.
 
-#### Fragment format
+#### Hash format
```

Comment:
Change needed: I'm splitting hairs here, but technically `#` is the hash and everything after the `#` is a fragment, per RFC 3986: https://tools.ietf.org/html/rfc3986#section-3.5